src/attr/_make.py

```diff
@@ -46,6 +46,19 @@
 _sentinel = object()
 
 
+if PY2 and object.__reduce__ is not int.__reduce__:
```

Comment:
pypy doesn't run under coverage because it's dead slow. You'll have to mark this for no coverage.